packages/rOpenScPCA/tests/testthat/test-sweep-clusters.R

```diff
@@ -0,0 +1,112 @@
+suppressPackageStartupMessages(library(SingleCellExperiment))
+
+set.seed(2024)
+sce <- splatter::simpleSimulate(nGenes = 1000, verbose = FALSE) |>
+  scater::logNormCounts() |>
+  scater::runPCA(ncomponents = 10)
+
+
+test_that("sweep_clusters works as expected with default algorithm & weighting", {
+  sweep_list <- sweep_clusters(
+    sce,
```

Comment:
since there is logic in the sweep function for the conversion, it is probably worth including a quick test that this function works for matrices and Seurat objects as well. I don't think you need to do anything but check that those functions run and produce output, so using full defaults (one clustering) should be fine.